torch/streambase.py

```diff
@@ -29,7 +29,7 @@ def __eq__(self, stream) -> bool:
         raise NotImplementedError
 
 
-class _EventBase(ABC):
+class EventBase(ABC):
     r"""Base Event class abstraction for multi backends Event to herit from"""
 
     @abstractmethod
```

Comment:
I'm not sure how I feel about dedicating all of torch.streambase for this.  What about putting it in torch.types?

Reply:
They are not type aliases

Reply:
@albanD Any opinion? I feel like it should go in the "device generic module", whatever that is.

Reply:
After checking the content of torch.types, I still don't think putting ABC inside there is a good idea because torch.types would expand with many imports and type alias.